dmrpp_generator/main.py

```diff
@@ -11,7 +11,7 @@ class DMRPPGenerator(Process):
     """
 
     def __init__(self, **kwargs):
-        self.processing_regex = '.*\\.(h(e)?5|nc(4)?)(\\.bz2|\\.gz|\\.Z)?'
+        self.processing_regex = '.*\\.(h(e)?5|H5|nc(4)?)(\\.bz2|\\.gz|\\.Z)?'
```

Comment:
Thank you for the PR.
Can we change this expression to 
`self.processing_regex = '.*\\.([h|H](e)?5|nc(4)?)(\\.bz2|\\.gz|\\.Z)?'` To have a somehow DRY expression?

Reply:
Than should be fine too. I almost did it that way, myself, so if you like that better I'm happy with it.